eunkyung/app.js

```diff
@@ -0,0 +1,53 @@
+const http = require('http')
+const express = require('express')
+const cors = require('cors')
+const dotenv = require('dotenv')
+const morgan = require('morgan')
+
+dotenv.config()
+
+const { DataSource } = require('typeorm')
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+})
+
+myDataSource.initialize()
+    .then(() => { console.log('data server has been initialized!') })
+
+const app = express()
+
+app.use(express.json())
+app.use(cors())
+app.use(morgan('tiny'))
+
+app.get('/ping', (req, res) => {
+    res.status(200).json({ message: 'pong' })
+})
+
+app.post('/sign_up', async (req, res, next) => {
+    const { name, password, email } = req.body
+    await myDataSource.query(
+        `INSERT INTO users (
+            name, 
+            password, 
+            email) 
+            VALUES (?,?,?);`,
+        [name, password, email]
+    )
+    res.status(201).json({ message: 'userCreated' })
+})
```

Comment:
코드의 가독성을 높이는 방법 중 들여쓰기를 통해 코드의 format을 관리하기 쉽게 작성하는 방법이 있습니다.
이를 위해서 indentation (들여쓰기)와 적절한 line breaking을 해주면 됩니다.

아래 예시를 참고해주세요:
```suggestion
app.post('/sign_up', async (req, res, next) => {
    const { name, password, email } = req.body
    await myDataSource.query(
        `INSERT INTO users (
            name, 
            password, 
            email
          ) 
          VALUES (
            ?,
            ?,
            ?
          );`,
        [name, password, email]
    )
    res.status(201).json({ message: 'userCreated' })
})
```